base/arrayshow.jl

```diff
@@ -410,14 +410,14 @@ function _show_nonempty(io::IO, X::AbstractMatrix, prefix::String)
                         show(io, el)
                     end
                 end
-                if last(cr) == last(indc)
+                if !isempty(cr) && last(cr) == last(indc)
```

Comment:
Would a more natural fix be to construct `cr2` as `first(indc) .+ (0:-1)`?

Reply:
That does sound like a better solution, I'll update this